testing_gateway.go

```diff
@@ -0,0 +1,42 @@
+package braintree
+
+// This action is only available in the sandbox environment.
```

Comment:
This comment should start with the types name for consistency with godoc.
e.g. `TestGateway exports actions only available in the sandbox environment.`